test/index.js

```diff
@@ -367,9 +367,9 @@ describe('integration', function(){
     });
 
     describe('when .options.events is an array', function(){
```

Comment:
Restructure these tests more clearly around the different types we intend for this to handle, which will give clarity to exactly what `map` can handle and what it can't